netcore/NetCore.Test/CacheFactoryUnitTest.cs

```diff
@@ -0,0 +1,97 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+using System;
+using Apache.Geode.NetCore;
+using Xunit;
+
+namespace GemfireDotNetTest
+{
+    [Collection("Geode .net Core Collection")]
+    public class CacheFactoryUnitTests
+    {
```

Comment:
In our contributing guide we should have a mention of the style guide use for .NET Core sources. We never derived one for the old .NET Framework C++/CLI sources because they were C++ based and loosely followed the C++ style where it was even possible to apply it. No tooling completely supported C++/CLI formatting so it is largely a mess. There is plenty of C# formatting tooling out there including `clang-format`. Since `clang-format` is already a requirement for C++ source formatting it make sense to extend it to C# sources with the agreed apon style applied.

Reply:
Works for me, though it does lend more urgency to the task to find a !@#$)*(&!@^$ version of `clang-format` that's consistent across platforms.

Reply:
I think adding in C# will have less urgency given that 99% of the work will likely be on Windows anyway. I also thing the variants in formatting are much fewer than C++ given the relative age ad popularity C++ has of C#.

Reply:
Added minimal `.clang-format` file and reformatted the code.  Using Google style guide by default, since we know from past queries that no one on the team has strong opinions re: style.